src/common/EmptyView/EmptyView.js

```diff
@@ -13,6 +13,7 @@ const EmptyView = ({
   docsUrl,
   description,
   urlLabel,
+  shouldBeHidden,
```

Comment:
But we change this in just one zerostate, so we would have to adjust all others if we choose your suggestion.   Am I wrong?